src/market/market.module.ts

```diff
@@ -140,11 +163,12 @@ export interface MarketModule {
    * Keep in mind that since this method returns an observable, nothing will happen until you subscribe to it.
    * Unsubscribing from the observable will stop the process and remove the demand from the market.
    */
-  startCollectingProposals(options: {
+  collectDraftOfferProposals(options: {
     demandSpecification: DemandSpecification;
     filter?: ProposalFilter;
-    bufferSize?: number;
-  }): Observable<OfferProposal[]>;
+    minProposalsBatchSize?: number;
+    proposalsBatchReleaseTimeoutMs?: number;
+  }): Observable<OfferProposal>;
```

Comment:
Note for reviewers: From the code flow the batching of draft proposals into array didn't bring any value. It was later just treated with a forEach and added to the pool without leveraging the batch at all.  I understand that by batching it was "throttling" the influx of draft proposals into the pool so that perhaps the pool could be fully utilized before next batch arrives. Nevertheless, I didn't see any added value here. Perhaps I missed something?